src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
+
+
+const maxFilesBrowsed = 5; // Maximum number of files to browse
```

Comment:
That could be done (wouldn’t update after resizing the browser without a high-CPU-load updates). Maybe make a dropdown for the user to select?